src/server/pfs/cmds/cmds_test.go

```diff
@@ -2,14 +2,33 @@ package cmds
 
 import (
 	"fmt"
+	"github.com/pachyderm/pachyderm/v2/src/server/pfs/fuse"
+	"strings"
 	"testing"
 
 	"github.com/pachyderm/pachyderm/v2/src/internal/minikubetestenv"
 	"github.com/pachyderm/pachyderm/v2/src/internal/require"
 	tu "github.com/pachyderm/pachyderm/v2/src/internal/testutil"
-	"github.com/pachyderm/pachyderm/v2/src/server/pfs/fuse"
 )
 
+func resourcesMap() map[string][]string {
```

Comment:
nit: I think we can just define resourcesMap & pluralsMap as const variables called `resources` and `plurals` if they aren't being modified

Reply:
I think with Go, we can't define const maps at a package scope (although that would be nice)